### DISCUSSION TITLE
Search should search things in tables​

### DISCUSSION THREAD

**Discussion by Macavirus** [toeverything/blocksuite#6582] [General]
### Description

Quick Search (Ctrl-K) should search text in tables as well.

### Use case

It is very common ot put text in a table, and later find that you want to find all all instances of text across all worksapces. The current global search does not do this.

Example:
In the sample content provided in v0.9 -- do a global search for "kingsolver". No results, yet this text exists in the "Personal Knowledge Management" page in the "8 books I should finished" table.

### Are you willing to submit a PR?

- [ ] Yes I'd like to help by submitting a PR!

**Comment by houjoe0829** [comment]
This should be caused by the current search index range not covering the contents of the Database. You can take a look. @hwangdev97

**Comment by hwangdev97** [comment]
Yes, we also want to support searching the content within the table. We will support related functions in subsequent iterations. Not just in the table, but also in the text information in edgeless
